cmd/generator/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/openshift/csi-operator/assets"
+	"github.com/openshift/csi-operator/assets/overlays/aws-ebs"
+	"github.com/openshift/csi-operator/pkg/generator"
+)
+
+// generator is a tool that generates assets for CSI driver operators.
+// It is intended to be used *before* building the operators, using `make update`.
+//
+// The generated assets will be then compiled into the operator binaries using assets.go.
+func main() {
+	// TODO: AWS EBS is hardcoded, add support for other CSI driver operators
+	flavour := flag.String("flavour", "standalone", "cluster flavour")
```

Comment:
`flavour` -> british English detected :joy: 

nit: add possible flavors to usage